hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveCatalog.java

```diff
@@ -1015,7 +1015,7 @@ public void testSnapshotStatsTableProperties() throws Exception {
   @Test
   public void testSetSnapshotSummary() throws Exception {
     Configuration conf = new Configuration();
-    conf.set("iceberg.hive.table-property-max-size", "4000");
+    long maxHiveTablePropertySize = 4000;
```

Comment:
can this be final?

Reply:
Yes it can. I found many variables in this class which are not final. Could you please create a ticket to fix those?

Reply:
While this comment is minor, and it could be ok to reject it, please consider that the reviewer spends his own time to review your code. Maybe it is not even in his job description, or in his task list to do it, so please be polite as much as possible.

Thanks,
Peter